document/core/syntax/instructions.rst

```diff
@@ -196,12 +200,12 @@ SIMD instructions provide basic operations over :ref:`values <syntax-value>` of
 
 .. math::
    \begin{array}{llcl}
-   \production{ishape} & \X{ixx} &::=&
+   \production{ishape} & \ishape &::=&
      \K{i8x16} ~|~ \K{i16x8} ~|~ \K{i32x4} ~|~ \K{i64x2} \\
-   \production{fshape} & \X{fxx} &::=&
+   \production{fshape} & \fshape &::=&
      \K{f32x4} ~|~ \K{f64x2} \\
-   \production{vshape} & \X{vxx} &::=&
-     \X{ixx} ~|~ \X{fxx} \\
+   \production{shape} & \shape &::=&
+     \ishape ~|~ \fshape \\
    \production{lane index} & \laneidx &::=& \byte \\
    \production{instruction} & \instr &::=&
      \dots \\&&|&
```

Comment:
[This comment applies to the line below, but I can't put it there.]
Where is `\vX{nnn}` defined? Shouldn't this be `\i128`?

Reply:
Ah yes, I think vnnn was defined previously and then remove, `\i128` is better here. Since we won't have any other type of values.